### DISCUSSION TITLE
Migration guide update / fix

### DISCUSSION THREAD

**Discussion by wencyen** [remix-run/react-router#9649] [v5 to v6 Migration] [answered]
Hi @ryanflorence and @timdorr,

In the migration guide linked [here](https://github.com/remix-run/react-router/discussions/8753), I believe there is an error in step 1. The `element` prop should be `component` instead. I'd like to request that this documentation be updated for other readers of this migration guide. Thank you.
```diff
// actual
- <Route path="/one/:two?" element={Comp} />
+ <CompatRoute path="/one/:two" element={Comp} />
+ <CompatRoute path="/one" element={Comp} />
```

```diff
// expected
- <Route path="/one/:two?" element={Comp} />
+ <CompatRoute path="/one/:two" component={Comp} />
+ <CompatRoute path="/one" component={Comp} />
```

**Comment by timdorr** [comment, marked as answer]
Please make a PR. That is the best way to get these kinds of things fixed.

**Reply by wencyen** [reply]
Thanks for getting back to me @timdorr. 

I don't believe the migration guide linked above is part of the repository. It looked to me a discussion thread that is written by @ryanflorence. If I am mistaken, I'd be happy to open a pull request but I just don't see how to if the file isn't part of the repository.

**Reply by timdorr** [reply]
Ah, my bad. Yeah, that can't be updated by PR. I fixed those lines now.

**Reply by wencyen** [reply]
thank you.
